### PR TITLE
feat(editor): Workflow history [WIP] - Remove pinned data from workflow history version preview (no-changelog)

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

**Review by krynble — COMMENTED** [review]
Given this is a new component, can we add a test to make sure that `pinData` is removed?

File: packages/editor-ui/src/components/__tests__/WorkflowPreview.test.ts [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+					command: 'openExecution',
+					executionId,
+					executionMode: '',
+				}),
+				'*',
+			);
+		});
+	});
+
+	it('should call also iframe postMessage with "setActiveExecution" is active execution is', async () => {
```
Comment (ivov) [inline packages/editor-ui/src/components/__tests__/WorkflowPreview.test.ts:142]: Typo?

File: packages/editor-ui/src/components/WorkflowPreview.vue [hunk 2]
```diff
@@ -8,202 +8,194 @@
...
+const ready = ref(false);
+const insideIframe = ref(false);
+const scrollX = ref(0);
+const scrollY = ref(0);
+
+const showPreview = computed(() => {
+	return (
+		!props.loading &&
+		((props.mode === 'workflow' && !!props.workflow) ||
+			(props.mode === 'execution' && !!props.executionId)) &&
```
[the remark this answers is not in the document]
Comment (cstuncsik) [reply packages/editor-ui/src/components/WorkflowPreview.vue]: was just copy pasted, fixed

File: packages/editor-ui/src/components/__tests__/WorkflowPreview.test.ts [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+		});
+
+		sendFakeIframeReadyMessage();
+
+		await waitFor(() => {
+			expect(postMessageSpy).not.toHaveBeenCalled();
+		});
+	});
+
+	it('should not call iframe postMessage when it is ready and there are no nodes in the workflow', async () => {
```
[the remark this answers is not in the document]
Comment (cstuncsik) [reply packages/editor-ui/src/components/__tests__/WorkflowPreview.test.ts:48]: This is fixed [here](https://github.com/n8n-io/n8n/pull/7434)

File: packages/editor-ui/src/components/__tests__/WorkflowPreview.test.ts (hunk 1 above)
Comment (cstuncsik) [reply packages/editor-ui/src/components/__tests__/WorkflowPreview.test.ts]: fixed

**Review by ivov — DISMISSED** [review]
Thanks for addressing everything!

**Review by ivov — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.14.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.14.0)

**Pull request merged**